app/controllers/users_controller.rb

```diff
@@ -0,0 +1,41 @@
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
+
+  end
+
+  def create
+    @user = User.new(strong_params)
+    if password_confirmation != true
+      render :new
+      flash[:notice] = "Those passwords don't match"
+    end
+    if @user.save
+      session[:user_id] = @user.id
+      flash[:notice] = "You are now registered and logged in."
+      redirect_to '/profile'
+    else
+      render :new
```

Comment:
Might be worth adding a flash message in here that says there is an error with their form/our servers

Reply:
@BrennanAyers opened an issue #91